index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
```

Comment:
what does these 3 meta tags do ?

Reply:
➡️ There is something called as character encoding. UTF-8 is basically the most updated and comprehensive character set through which you can insert most of the characters and symbols popularly used around the globe.
➡️ By default mobile browsers squashes the desktop content into mobile size which makes it difficult to read, thus to overwrite the default viewport we use meta tag.
➡️ X-UA-Compatible allows us to choose the version of Internet Explorer the page should be rendered as, here it is edge.